pyproject.toml

```diff
@@ -40,49 +40,48 @@ documentation = "https://pypi.org/project/microbootstrap/"
 
 [tool.poetry.dependencies]
 python = "^3.9"
-pydantic-settings = "^2.3.4"
-granian = { version = "^1.4.4", extras = ["reload"] }
-typing-extensions = "^4.12.2"
-rich = "^13.7.1"
-eval-type-backport = "^0.2.0"
+pydantic-settings = "*"
```

Comment:
I'd keep lower bound constrains

Reply:
you mean for every package? seems like it will block some old versions from using the library even though it can work potentially

Reply:
Yes, but then we don't have to think about supporting very old versions